test/app-luatest/gh_9806_http_client_version_test.lua

```diff
@@ -0,0 +1,100 @@
+local t = require('luatest')
+local treegen = require('test.treegen')
+local justrun = require('test.justrun')
+local socket = require('socket')
+
+local g = t.group()
+
+g.before_all(treegen.init)
+g.after_all(treegen.clean)
+
+g.before_each(function(g)
+   g.server = socket.tcp_server('127.0.0.1', 0, function(_s) end)
+end)
+
+g.after_each(function(g)
+   g.server:close()
+end)
```

Comment:
Nit: three spaces indent (instead of 4).

Reply:
Fixed, but luacheck didn't say anything